docs/user_guide.rst

```diff
@@ -170,16 +170,73 @@ Now we can study how the circuit gets transpiled:
                                                                                               ║
            meas_2: ═══════════════════════════════════════════════════════════════════════════╩═
 
-We can also simulate the execution of the transpiled circuit before actually executing it:
+
+Simulating the execution of a transpiled circuit locally
+--------------------------------------------------------
+
+The execution of any transpiled circuit can be simulated locally, before actually executing it on
+real hardware, by executing it on an :class:`.IQMFakeBackend` instance: 
 
 .. code-block:: python
 
-    from qiskit import Aer
+    from qiskit import execute, QuantumCircuit
+    from qiskit_iqm import IQMFakeAdonis
+    from qiskit.compiler import transpile
+
+    circuit = QuantumCircuit(2)
+    circuit.h(0)
+    circuit.cx(0, 1)
+    circuit.measure_all()
+
+    backend = IQMFakeAdonis()
+    circuit = transpile(circuit, backend=backend)
```

Comment:
you don't need to transpile manually, the `execute` function will take care of it